src/AssetManager.Core/Entities/Requests.cs

```diff
@@ -11,14 +11,10 @@ namespace AssetManager.Core.Entities
 {
     public class Requests: Entity
     {
-        [ForeignKey("AssetId")]
+        [ForeignKey("Asset")]
         public int AssetId { get; set; }
         public virtual Asset Asset { get; set; }
-
-        [ForeignKey("UserId")]
         public int UserId { get; set; }
-        public virtual User User { get; set; }
```

Comment:
I think you have missed the `ForeignKey` attribute. Although Entity Framework is smart enough to understand the connection. 

You can also use 
```csharp
public int UserId { get ; set; }
[ForeignKey("UserId")]
public virtual User User { get; set; }
```

Reply:
no need now. i will add this in the next work

Reply:
@kajal-alpha : Okay.